www/src/components/LeaveFeedback.tsx

```diff
@@ -1,29 +1,57 @@
-import React from 'react';
+import React, { AnchorHTMLAttributes } from 'react';
 import PropTypes from 'prop-types';
-import { Nav } from '~paragon-react';
+import { Button, Hyperlink } from '~paragon-react';
 import { useLocation } from '@reach/router';
 
-function LeaveFeedback(props) {
+export interface LeaveFeedbackProps extends Partial<AnchorHTMLAttributes<HTMLAnchorElement>> {
+  isLink?: boolean;
+}
+
+function LeaveFeedback({ isLink, ...props }: LeaveFeedbackProps) {
   const location = useLocation();
   const FEEDBACK_URL = `https://github.com/openedx/paragon/issues/new?title=%5Bparagon-openedx.netlify.app%5D%20Feedback%20(on%20${location.pathname})&amp;labels=docs&template=feedback_template.md`;
+  const leaveFeedbackLinkTitle = 'Leave feedback';
 
   const handleLinkFeedbackClick = () => {
     global.analytics.track('openedx.paragon.docs.leave_feedback.clicked');
   };
 
+  if (isLink) {
```

Comment:
I think it also make more sense to leave `Nav.Link` implementation here since we only use it inside `Nav` component, e.g.
```javascript
if (isNavLink) {
  return (
    <Nav.Link
      onClick={handleLinkFeedbackClick}
      href={FEEDBACK_URL}
      target="_blank"
      rel="noopener noreferrer"
      {...props}
    >
      {leaveFeedbackLinkTitle}
    </Nav.Link>
  )
}
```

Reply:
Done